icaruscode/CRT/CRTUtils/CRTHitRecoAlg.cc

```diff
@@ -441,16 +322,17 @@ vector<pair<sbn::crt::CRTHit, vector<int>>> CRTHitRecoAlg::CreateCRTHits(
   }        // loop over side CRTData products
 
   if (fVerbose) {
-    mf::LogInfo("CRT") << returnHits.size() << " CRT hits produced!" << '\n'
-                       << "  nHitC: " << nHitC << " , nHitD: " << nHitD
-                       << " , nHitM: " << nHitM << '\n'
-                       << "  nMisC: " << nMissC << " , nMisD: " << nMissD
-                       << " , nMisM: " << nMissM << '\n';
+    mf::LogInfo("CRTHitRecoAlg: ")
+      << returnHits.size() << " CRT hits produced!" << '\n'
+      << "  nHitC: " << nHitC << " , nHitD: " << nHitD
+      << " , nHitM: " << nHitM << '\n'
+      << "  nMisC: " << nMissC << " , nMisD: " << nMissD
+      << " , nMisM: " << nMissM << '\n'
+      << " CRT Hits by region" << '\n';
     auto cts = regCounts.begin();
-    mf::LogInfo("CRT") << " CRT Hits by region" << '\n';
     while (cts != regCounts.end()) {
-      mf::LogInfo("CRT") << "reg: " << (*cts).first
-                         << " , hits: " << (*cts).second << '\n';
+      std::cout  << "reg: " << (*cts).first
```

Comment:
Are you sure you want this as a std::cout and not as a Log Message?
Just wanna make sure it does not overflow the console with print messages